### PR TITLE
Added Python tool for sorting out the Json changes and providing instructions and moves addressable bundles to github

### PR DESCRIPTION
**Pull request by Bod9001**
Added Python tool for sorting out the Json changes and providing instructions

pretty cool tool

### CODE REVIEW

File: Tools/python addressable prepare for upload/addressable_prepare_for_upload.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    catalog_file = None
+    for file in os.listdir(destination_path):
+        if file.endswith(".json"):
+            catalog_file = file
+
+    print(f"Creating txt file for {bundle_name}...")
+    with open(destination_path / f"{bundle_name}.txt", "w") as f:
+        if not catalog_file:
+            print(f"No catalog file found for {bundle_name}!")
```
Comment (Bod9001) [inline Tools/python addressable prepare for upload/addressable_prepare_for_upload.py]: Why not just make the file?

Comment (Bod9001) [reply Tools/python addressable prepare for upload/addressable_prepare_for_upload.py]: like I don't want to be messing around with making new txt When new addressable project is being added

Comment (corp-0) [reply Tools/python addressable prepare for upload/addressable_prepare_for_upload.py]: not the txt, the json catalog file. You can't make that manually.

Comment (Bod9001) [reply Tools/python addressable prepare for upload/addressable_prepare_for_upload.py]: oh ok, was confused by the with open(destination_path / f"{bundle_name}.txt", "w") as f: Right above it

Comment (corp-0) [reply Tools/python addressable prepare for upload/addressable_prepare_for_upload.py]: fair I could probably move it above the open

File: Tools/python addressable prepare for upload/addressable_prepare_for_upload.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            if "AddressablePackingProjects" in entry:
+                bundle_file = Path(entry).name
+                json_data["m_InternalIds"][i] = f"{GITHUB_CONTENT_URL}/{bundle_name}/{bundle_file}"
+            i += 1
+        json.dump(json_data, f)
+
+def finishing_tasks():
+    print("The following bundle folders were created:")
+    for bundle in os.listdir(RESULT_CONTENT_PATH):
+        print(f"- {bundle}")
```
Comment (Bod9001) [inline Tools/python addressable prepare for upload/addressable_prepare_for_upload.py:88]: should also probably log the path to the txt files,  so you can just copy and paste after making if you're adding a new one

Comment (corp-0) [reply Tools/python addressable prepare for upload/addressable_prepare_for_upload.py:88]: okay

**Review by corp-0 — APPROVED** [review]
I don't think you will be able to merge if I don't approve it first

**Pull request merged**